src/chimera/instruments/telescope.py

```diff
@@ -50,6 +52,12 @@ def slewToObject(self, name):
         self.slewToRaDec(target)
 
     @lock
+    def slewToSolarSystemObject(self, name):
+        target = SsODNetlookup(name)
+        print "telescope got from SsODNet "+str(target)
```

Comment:
This can be interesting. Add the feature of slewing to solar systems on chimera-tel.